Source/BLE.Client/BLE.Client.Maui/Platforms/Android/AndroidManifest.xml

```diff
@@ -1,18 +1,23 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android" package="com.companyname.ble.client.maui" android:versionCode="1" android:versionName="1.0">
-	<application android:allowBackup="true" android:supportsRtl="true" android:label="BLE.Client.Maui"></application>
+	<application android:allowBackup="true" android:supportsRtl="true" android:label="BLE.Client.Maui"
+				 android:icon="@mipmap/appicon" android:roundIcon="@mipmap/appicon_round"
+				 ></application>
 	<uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
 	<uses-permission android:name="android.permission.INTERNET" />
-	<uses-permission android:name="android.permission.BLUETOOTH_ADMIN" />
-	<uses-permission android:name="android.permission.BLUETOOTH" />
-	<uses-permission android:name="android.permission.ACCESS_COARSE_LOCATION" />
-	<uses-permission android:name="android.permission.ACCESS_FINE_LOCATION" />
-	<uses-permission android:name="android.permission.ACCESS_LOCATION_EXTRA_COMMANDS" />
-	<uses-permission android:name="android.permission.ACCESS_MOCK_LOCATION" />
-	<uses-permission android:name="android.permission.BLUETOOTH_ADVERTISE" />
-	<uses-permission android:name="android.permission.BLUETOOTH_CONNECT" />
-	<uses-permission android:name="android.permission.BLUETOOTH_PRIVILEGED" />
+
+	<!-- See https://developer.android.com/develop/connectivity/bluetooth/bt-permissions
+		 for more information about which permissions to use for your target platform
+	-->
+	<uses-feature android:name="android.hardware.bluetooth_le" android:required="true" />
 	<uses-permission android:name="android.permission.BLUETOOTH_SCAN" android:usesPermissionFlags="neverForLocation" />
-	<uses-permission android:name="android.permission.CHANGE_NETWORK_STATE" />
-	<uses-permission android:name="android.permission.CONTROL_LOCATION_UPDATES" />
+	<uses-permission android:name="android.permission.BLUETOOTH_CONNECT" />
+	<!-- Needed for Android 11 and lower -->
+	<uses-permission android:name="android.permission.BLUETOOTH" android:maxSdkVersion="30" />
+	<uses-permission android:name="android.permission.ACCESS_FINE_LOCATION" android:maxSdkVersion="30" />
```

Comment:
Alright, this looks much better now. I verified that the app runs on an Android 11 device again. It properly asks for the location permission there.